rest-api/src/main/resources/scheduling/oozie/workflow_template.xml

```diff
@@ -32,7 +32,7 @@
                 --num-executors $stdNumExecutors
                 --executor-memory $stdExecutorMemory
                 --driver-cores $driverCores
-                --conf $sparkConfQuotesspark.driver.extraJavaOptions=-Dmenas.rest.uri='$menasRestURI' -Dspline.producer.url='$lineageWriteApiUrl' -Dlog4j.configuration='spark-log4j.properties'$sparkConfQuotes $splineMode
+                --conf $sparkConfQuotesspark.driver.extraJavaOptions=-Denceladus.rest.uri='$menasRestURI' -Dspline.producer.url='$lineageWriteApiUrl' -Dlog4j.configuration='spark-log4j.properties'$sparkConfQuotes $splineMode
```

Comment:
```suggestion
                --conf $sparkConfQuotesspark.driver.extraJavaOptions=-Denceladus.rest.uri='$enceladusRestURI' -Dspline.producer.url='$lineageWriteApiUrl' -Dlog4j.configuration='spark-log4j.properties'$sparkConfQuotes $splineMode
```

Feel free to leave this one for the subsequent PR that solves the field names if you feel it rather belongs there.